src/coingecko/Coingecko.ts

```diff
@@ -111,7 +117,7 @@ export class Coingecko {
         const { host } = this;
         const url = `${host}/${path}`;
         const result = await axios(url, { params: { x_cg_pro_api_key: this.apiKey } });
-        console.log(`INFO(coingecko): Sent GET request to url ${result.request.responseURL}`);
+        this.logger.info("sdk-v2/coingecko", `Sent GET request to url ${result.request.responseURL}`, {});
```

Comment:
You should use the standard winston format with `{ at: ..., message: ...}` etc

Reply:
Updated.